src/styles/colours.module.css

```diff
@@ -1,8 +1,11 @@
 :root {
   --lightBlue: #3498db;
   --blue: #2980b9;
+  --darkBlue: rgb(2, 2, 117);
+  --darkerBlue: rgb(2, 2, 91);
   --lightPurple: #b393d3;
+  --darkPurple: rgb(75, 4, 75);
   --white: rgb(255, 255, 255);
-  --grey: #ddd;
+  --grey: #ccc;
   --black: rgba(0, 0, 0);
 }
```

Comment:
you have a set of variables. Can you overwrite them with different colors depending on the theme? The way you have it set up currently is not very extendable. What if you wanted to have more palettes - how easy would it be to add them?